dev/devicelab/lib/framework/ios.dart

```diff
@@ -116,14 +116,14 @@ Future<void> testWithNewIOSSimulator(
 }
 
 /// Shuts down and deletes simulator with deviceId.
-Future<void> removeIOSimulator(String deviceId) async {
+Future<void> removeIOSimulator(String? deviceId) async {
```

Comment:
as this function handles `null`, I believe the root problem was that when this file was migrated, this parameter type should have been made nullable.

Reply:
I am wondering why the post submit run has never hit this flake: https://ci.chromium.org/p/flutter/builders/prod/Mac%20module_test_ios?limit=100?

Reply:
maybe post-submit has never failed early enough?